emf/multipkg.json

```diff
@@ -92,6 +110,14 @@
         "$ref" : "/0/Statement"
       },
       "containment" : true
+    }, {
```

Comment:
And here

Reply:
Actually that file should have been ignored, my mistake